elixir/apps/web/lib/web/live/actors/components.ex

```diff
@@ -125,6 +125,27 @@ defmodule Web.Actors.Components do
     """
   end
 
+  def provider_form(%{provider: %{adapter: :openid_connect}} = assigns) do
+    ~H"""
+    <div>
+      <.input
+        label="Email or Sub claim"
+        placeholder="Email or Sub claim"
+        field={@form[:provider_identifier]}
+        autocomplete="off"
+      />
```

Comment:
```suggestion
      />
      <p class="mt-2 text-xs text-neutral-500">
        The user's email or <code>sub</code> claim value. This will be used to match the user to this identity when signing in.
      </p>
```